clientlib/src/main/proto/yelp/nrtsearch/search.proto

```diff
@@ -251,6 +251,19 @@ message Query {
     }
 }
 
+message PluginRescorer {
+    int32 windowSize = 1;
+    Script script = 2;
```

Comment:
I would suggest just adding the needed fields here instead of using the Script message, since this isn't really a script. You could probably get away with using a google.protobuf.Struct for parameters, since typing shouldn't be a big issue.

Reply:
Okok, I used `google.protobuf.Struct` for params when defining PluginRescorer